src/main/webapp/style.css

```diff
@@ -4,4 +4,4 @@ h3 {
 
 .srp-container {
   display: flex;
-}
+}
```

Comment:
new line

Reply:
was solving these issue when you're commenting lol